### PR TITLE
[feat]Ascend NPU Gemma-3-12b and Gemma-3-27b support

### PR DESCRIPTION
**Pull request by VDV1985**
## Motivation

Support of Gemma-3-12b and Gemma-3-27b on NPU, return back NPU graph tests

## Modifications

Attention extend modified to support qk_head_dim > 128, NPU specific implementations for some activation functions, NPU specific implementations for Gemma RMSNorm, NPU specific apply_rotary_pos_emb implementation, fast image processor for multimodal models enabled on NPU, NPU graph tests were missed because of tests redesign, tests were placed into correct place and enabed back.

## Accuracy Test

MMMU test for gemma-3b-27-it - 0.51

## Checklist

- [ ] Format your code according to the [Code Formatting with Pre-Commit](https://docs.sglang.ai/references/contribution_guide.html#code-formatting-with-pre-commit).
image

- [ ] Add unit tests as outlined in the [Running Unit Tests](https://docs.sglang.ai/references/contribution_guide.html#running-unit-tests-adding-to-ci).
- [ ] Update documentation / docstrings / example tutorials as needed, according to [Writing Documentation](https://docs.sglang.ai/references/contribution_guide.html#writing-documentation-running-docs-ci).
- [ ] Provide throughput / latency benchmark results and accuracy evaluation results as needed, according to [Benchmark and Profiling](https://docs.sglang.ai/references/benchmark_and_profiling.html) and [Accuracy Results](https://docs.sglang.ai/references/accuracy_evaluation.html).
- [ ] For reviewers: If you haven't made any contributions to this PR and are only assisting with merging the main branch, please remove yourself as a co-author when merging the PR.
- [ ] Please feel free to join our Slack channel at https://slack.sglang.ai to discuss your PR.

### CODE REVIEW

**Review by JustinTong0323 — APPROVED** [review]
mm part LGTM

File: python/sglang/srt/managers/mm_utils.py [hunk 1]
```diff
@@ -20,9 +20,11 @@
 )
 from sglang.srt.mem_cache.multimodal_cache import MultiModalCache
 from sglang.srt.model_executor.forward_batch_info import ForwardBatch
-from sglang.srt.utils import flatten_nested_list, print_warning_once
+from sglang.srt.utils import flatten_nested_list, is_npu, print_warning_once
 from sglang.utils import logger
 
+_is_npu = is_npu()
```
Comment (mickqian) [inline python/sglang/srt/managers/mm_utils.py:26]: Can we move to somewhere like `utils`?

Comment (VDV1985) [reply python/sglang/srt/managers/mm_utils.py:26]: What do you mean? "\_is_{device} = is_{device}()" lines are used all over the code

File: python/sglang/srt/layers/rotary_embedding.py [hunk 2]
```diff
@@ -1882,20 +1882,32 @@ def apply_rotary_pos_emb(
     sin: torch.Tensor,
     unsqueeze_dim=1,
 ) -> Tuple[torch.Tensor, torch.Tensor]:
-    orig_q_dtype = q.dtype
-    orig_k_dtype = k.dtype
-    q, k = q.float(), k.float()
+    if not _is_npu:
```
Comment (Alcanderian) [inline python/sglang/srt/layers/rotary_embedding.py]: suggestion
```
def apply_rotary_pos_emb():
    ...

def apply_rotary_pos_emb_npu():
    ...

if _is_npu:
    apply_rotary_pos_emb = apply_rotary_pos_emb_npu
```

Comment (VDV1985) [reply python/sglang/srt/layers/rotary_embedding.py]: done

File: python/sglang/srt/layers/layernorm.py [hunk 3]
```diff
@@ -259,23 +259,46 @@ def forward_cuda(
...
+        return x if residual is None else (x, residual)
+
+class Gemma3RMSNorm(CustomOp):
     def __init__(self, dim: int, eps: float = 1e-6):
         super().__init__()
         self.eps = eps
         self.weight = nn.Parameter(torch.zeros(dim))
+        # Re-dispatch
+        if not _is_npu:
+            self._forward_method = self.forward_native
```
[the remark this answers is not in the document]
Comment (VDV1985) [reply python/sglang/srt/layers/layernorm.py]: done

File: python/sglang/srt/managers/mm_utils.py (hunk 1 above)
Comment (mickqian) [reply python/sglang/srt/managers/mm_utils.py:26]: I mean, if we can reuse the same global variable

**Review by Alcanderian — APPROVED** [review]
